### PR TITLE
Update spinq10q connection

### PR DESCRIPTION
**Pull request by PiergiorgioButtarini**
This PR update the runcard of spinq10q  in which only the first 5 qubits are connected to Qblox drivers.

### CODE REVIEW

**Review by GabrielePalazzo — APPROVED** [review]

File: spinq10q/parameters.json [hunk 1]
```diff
@@ -1,859 +1,402 @@
...
+  "settings": {
+    "nshots": 1024,
+    "relaxation_time": 20000
+  },
+  "qubits": [
+    1,
+    2,
+    3,
+    4,
+    5
```
Comment (andrea-pasquale) [inline spinq10q/parameters.json:12]: Just to understand, if this configuration is for the first 5 qubits shouldn't it be from 0 to 4?
Or is there a qubit 0 which we are not considering

File: spinq10q/platform_old.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import pathlib
```
[the remark this answers is not in the document]
Comment (alecandido) [reply spinq10q/platform_old.py:1]: If there is no chance for them to be reused, just remove them.

File: spinq10q/parameters.json (hunk 1 above)
Comment (PiergiorgioButtarini) [reply spinq10q/parameters.json]: Thanks for spotting this, solved in https://github.com/qiboteam/qibolab_platforms_qrc/pull/114/commits/e20735cbd0c2a7e74339664331a26d62b9ae5e39!

**Review by GabrielePalazzo — APPROVED** [review]

File: spinq10q/platform_old.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (PiergiorgioButtarini) [reply spinq10q/platform_old.py]: I removed the old files regarding spinq10q in https://github.com/qiboteam/qibolab_platforms_qrc/pull/114/commits/e20735cbd0c2a7e74339664331a26d62b9ae5e39. Regarding the other platforms I don't really know which are the outdated ones so I prefer to keep them.

**Comment by PiergiorgioButtarini** [comment]
Thanks @biankawolo for this runcard! You can check the comments and the requested changes made by the other guys in order to learn for future PRs.

File: spinq10q/platform_old.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrea-pasquale) [reply spinq10q/platform_old.py]: Thanks @PiergiorgioButtarini.
Let me open an issue for the outdated platforms.

**Review by andrea-pasquale — APPROVED** [review]

**Review by biankawolo — APPROVED** [review]

File: spinq10q/parameters.json (hunk 1 above)
Comment (biankawolo) [reply spinq10q/parameters.json]: si we enumerate qubits from 0 always?

Comment (andrea-pasquale) [reply spinq10q/parameters.json]: In programming, especially in Python, this convention is used.

**Pull request merged**